src/Microsoft.Identity.Client/Utils/ScopeHelper.cs

```diff
@@ -64,5 +64,16 @@ internal static SortedSet<string> CreateSortedSetFromEnumerable(IEnumerable<stri
             }
             return new SortedSet<string>(input);
         }
+
+        public static string ConvertSortedSetScopesToString(SortedSet<string> singleString)
```

Comment:
> ConvertSortedSetScopesToString(Sor [](start = 28, length = 35)

I believe there is already somewhere an extension method or similar which does this ? AsSingleString  or ?